nodes.lua

```diff
@@ -0,0 +1,393 @@
+                 }
+      },
+    },
+
+    { suffix = "base",
+      model  = "mymillwork_mach19.obj",
+      mdesc  = "Baseboard",
+      sbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.4375, 0.5, -0.1875, 0.5},
+                 }
+      },
+      cbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.4375, 0.5, -0.1875, 0.5},
+                 }
+      },
+    },
+
+    { suffix = "base_ic",
+      model  = "mymillwork_mach20.obj",
+      mdesc  = "Baseboard IC",
+      sbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.4375, 0.5, -0.1875, 0.5},
+                          {-0.5, -0.5, -0.5, -0.4375, -0.1875, 0.4375},
+                 }
+      },
+      cbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.4375, 0.5, -0.1875, 0.5},
+                 }
+      },
+    },
+
+    { suffix = "base_oc",
+      model  = "mymillwork_mach21.obj",
+      mdesc  = "Baseboard OC",
+      sbox   = { type = "fixed",
+                 fixed = {{0.4375, -0.5, 0.4375, 0.5, -0.1875, 0.5},
+                 }
+      },
+      cbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.4375, 0.5, -0.1875, 0.5},
+                 }
+      },
+    },
+
+    { suffix = "base_fancy",
+      model  = "mymillwork_mach22.obj",
+      mdesc  = "Fancy Baseboard",
+      sbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.3125, 0.5, 0.1875, 0.5},
+                 }
+      },
+      cbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.3125, 0.5, 0.1875, 0.5},
+                 }
+      },
+    },
+
+    { suffix = "base_fancy_ic",
+      model  = "mymillwork_mach23.obj",
+      mdesc  = "Fancy Baseboard IC",
+      sbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.3125, 0.5, 0.1875, 0.5},
+                          {-0.5, -0.5, -0.5, -0.3125, 0.1875, 0.3125},
+                 }
+      },
+      cbox   = { type = "fixed",
+                 fixed = {{-0.5, -0.5, 0.3125, 0.5, 0.1875, 0.5},
+                          {-0.5, -0.5, -0.5, -0.3125, 0.1875, 0.3125},
+                 }
+      },
+    },
+
+    { suffix = "base_fancy_oc",
+      model  = "mymillwork_mach24.obj",
+      mdesc  = "Fancy Baseboard OC",
+      sbox   = { type = "fixed",
+                 fixed = {{0.3125, -0.5, 0.3125, 0.5, 0.1875, 0.5},
+                 }
+      },
+      cbox   = { type = "fixed",
+                 fixed = {{0.3125, -0.5, 0.3125, 0.5, 0.1875, 0.5},
+                 }
+      },
+    },
+
+}
+
+function mymillwork.register(material, node_suffix, desc, image, group)
+
+    for __, data in ipairs(mymillwork.nodes) do
+        mymillwork.register_all(data.suffix, data.model, data.mdesc, data.sbox, data.cbox,
+                             node_suffix, material, desc, image, group)
+    end
+
+end
+
+function mymillwork.register_all(suffix, model, mdesc, sbox, cbox, node_suffix, material, desc, image, group)
```

Comment:
The previous definition was
```Lua
function mymillwork.register_all(mat, desc, image, group, itm)
```
which means that the new API function now has a different behaviour and possibly causes a crash when another mod wants to use the old method.